### PR TITLE
feat: combine p-select-dropdown and p-filterable-dropdown

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- SSIA

Updated P-Select-Dropdown Spec
- ...basic P-Select-Dropdown Spec (like disabled...)
- ...basic P-Filterable-Dropdown's spec (like handler...)
- StyleType
image

- AppearanceType
image

- SelectionLabel
image
image

- SelectionHighlight
image

- ShowAlertDot
image

- ShowDeleteAllButton
image

- IsFilterable
image

### CODE REVIEW

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue [hunk 1]
```diff
@@ -1,88 +1,106 @@
...
-
-        return null;
+    proxyVisibleMenu: useProxyValue<boolean>('visibleMenu', props, emit),
+    proxySelectedItem: useProxyValue<SelectDropdownMenuItem[]>('selected', props, emit),
+    proxySearchText: useProxyValue('searchText', props, emit),
+    showDeleteAllButton: computed(() => {
+        if (!props.showDeleteAllButton) return false;
+        if (props.disabled) return false;
+        if (props.readonly) return false;
+        return !!state.proxySelectedItem.length;
```
Comment (WANZARGEN) [inline packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:103]: praise: very clear

**Review by WANZARGEN — APPROVED** [review]
LGTM!! I left very tiny nitpick comments and some suggestion!

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue [hunk 2]
```diff
@@ -1,88 +1,106 @@
...
+    showAlertDot?: boolean;
+    showDeleteAllButton?: boolean;
+    useFixedMenuStyle?: boolean;
     buttonIcon?: string;
-    menuPosition?: CONTEXT_MENU_POSITION;
-    readOnly?: boolean;
-    size?: SelectDropdownSize;
     isFixedWidth?: boolean;
+
+    /* context menu props */
```
Comment (yuda110) [inline packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:42]: praise: nice comment!!!

File: packages/mirinae/src/inputs/dropdown/select-dropdown/components/dropdown-button.vue [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+    buttonIcon?: string;
+    invalid?: boolean;
+    disabled?: boolean;
+    placeholder?: string;
+    selectionLabel?: string;
+    showAlertDot?: boolean;
+    showDeleteAllButton?: boolean;
+    isFixedWidth?: boolean;
+    selectionHighlight?: boolean;
+    visibleMenu?: boolean;
```
[the remark this answers is not in the document]
Comment (yuda110) [reply packages/mirinae/src/inputs/dropdown/select-dropdown/components/dropdown-button.vue:30]: `visibleTargetMenu`...? 🧐

**Review by yuda110 — APPROVED** [review]
LGTM!!!!!!! Thank youuuuu

**Pull request merged**